assets/scss/vars.scss

```diff
@@ -1,12 +1,11 @@
-@import "~bulma/sass/utilities/_all";
-
 // Brand Colours
 $brand-pink: #E30285;
 $brand-turquoise: #008080;
 
 // Background Colours
 $background-dark: #020202;
 $background-medium: #2C2C3E;
+$background-app: #181829;
```

Comment:
Could you show me where this comes from? To my knowledge I thought the background should be the same colour on desktop and mobile. Maybe I am missing a design or something?

Reply:
This is the original colour we were using for the background for multiple elements on this web page before Vanessa told me we wanted the background to be our brand black to make the video iframe look better. In the bootbag mobile app repo, if you search this hex code, we call it "Mirage" which we use for backgroundDark.

Reply:
Is the background colour on mobile shown in one of the videos? I am struggling to see the difference in the background colours on the videos.

I believe this would also trigger for web mobile users too? I think we might need to rethink the variable name as it sort of implies it is only that colour if you come to the web version via the app.